components/Home.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  ViewStyle,
+  TextStyle,
+  StyleProp,
+  TouchableOpacity,
+  ImageStyle
+} from 'react-native';
+
+interface IProps {
```

Comment:
what's IProps? Change to something more appropriate

Reply:
IProps is interface props. That is validating props for the component. Will changing IProps to interfaceProps be better?